scripts/node-unit-tests.ts

```diff
@@ -29,14 +35,18 @@ const NODE_8_LEGACY_DEPENDENCIES = [
 ];
 
 const NODE_10_SKIP_TESTS_PACKAGES = ['@sentry/remix', '@sentry/sveltekit'];
-const NODE_10_LEGACY_DEPENDENCIES = ['jsdom@16.x', 'lerna@3.13.4'];
+const NODE_10_LEGACY_DEPENDENCIES = [
+  'jsdom@16.x',
+  'lerna@3.13.4',
+  'jest-environment-jsdom@25.x',

```

Comment:
Let's see, according to https://github.com/jsdom/jsdom/issues/2961 this is also needed for node 10, got failing test there that jsdom is not working 😬